src/components/Menu.js

```diff
@@ -37,7 +37,7 @@ class Menu extends React.Component {
                 <Link to="/helplines">Helplines</Link>
               </NavItem>
               <NavItem>
-                <NavLink href="./components/Guide">How To Use Guide</NavLink>
+                <NavLink href="guide">How To Use Guide</NavLink>
```

Comment:
Swith with Link rather than Navlink and use "/guide" over "guide"

Reply:
👍 with Bill. - Beat me to it